basilisk/gui/ocr_handler.py

```
@@ -0,0 +1,336 @@
+			return
+
+		if not attachment_files:
+			wx.MessageBox(
+				# Translators: This message is displayed when there are no attachments to perform OCR on.
+				_("No attachments to perform OCR on."),
+				_("Error"),
+				wx.OK | wx.ICON_ERROR,
+			)
+			return
+
+		if not self.parent._check_attachments_valid():
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 8

(W0311)